webapp/src/app.tsx

```diff
@@ -66,18 +66,18 @@ const App = React.memo((): JSX.Element => {
                             <Route path='/change_password'>
                                 <ChangePasswordPage/>
                             </Route>
-                            <Route path='/shared'>
+                            <Route path='/shared/:boardId?/:viewId?'>
```

Comment:
@chenilim I'm changin this to simplify the approach and allow to move that logic into the router, like any other param. Do you see any problem on that (it is probably breaking the old links, I can try to make it backward compatible).

Reply:
Sounds good. Please add code to redirect the old format to the new one, and a note in the changelog of this breaking change.

Reply:
Done.